app/generic/verifier/facades/acapy_verifier.py

```diff
@@ -87,17 +92,74 @@ async def reject_proof_request(
         cls, controller: AcaPyClient, proof_request: RejectProofRequest
     ) -> None:
         """
-        Accept proof request
+        Reject proof request
 
         Parameters:
         -----------
         controller: AcaPyClient
             The aries_cloudcontroller object
-        proof_request: AcceptProofRequet
+        proof_request: RejectProofRequet
             The proof request object
 
+        Returns:
+        --------
+        None
+            Returns None on successful request rejection.
+        """
+
+    @classmethod
+    @abstractmethod
+    async def delete_proof(cls, controller: AcaPyClient, proof_id: str) -> None:
+        """
+        Delete proof request
+
+        Parameters:
+        -----------
+        controller: AcaPyClient
+            The aries_cloudcontroller object
+        pres_ex_id: str
+            The proof record exchange id
+
         Returns:
         --------
         None
             Returns None on successful record deletion.
         """
+
+    @classmethod
+    @abstractmethod
+    async def proofs(
```

Comment:
To be consistent with issuer (or also change the issuer class `get_records` is not the best name prob.). But consistency is key :)

```suggestion
    async def get_records(
```